README.md

```diff
@@ -62,6 +62,36 @@ cd ~/go/src/spa-website-2022 && make post-wsl
 spa # alias for cd ~/go/src/spa-website-2022 set in .zshrc or .bashrc
 ```
 ---
+### __GUM__:
+* GUM can be used for the following: 
+    * ``Run Client (requests towards local API)``
+    * ``Run Client (requests towards deployed API)``
+    * ``Run API``
+    * ``SSH into a VM``
+    * ``Set up VM for Deployment``
+
+*If you don't want to use gum, you may proceed to the next sections*
+#### __Set up__:
+```
+1. vim ~/.bashrc
+2. Add export HUB_VM="root@188.166.65.120" (make sure it's not in an if statement or for cycle)
```

Comment:
Remember what we discussed to for the IPs? Don't put them here.

Reply:
I thought of this, but how to I tell them what to put as a variable?